app/views/application/_navigation.html.slim

```diff
@@ -14,4 +14,7 @@ nav.navbar.navbar-top
       ul.nav.navbar-nav
         - if current_user.present?
           li
+            = link_to 'Create superadmin', new_user_url
```

Comment:
`link_to Users' , users_path` and delete the other 2 links, the should be added to the users index page